docs/storage/index.md

```diff
@@ -72,9 +69,14 @@ The use of storage is billed according to the [billing policy](../runjobs/lumi_e
     billing of the project scratch and project flash as detailed on the
     [billing page][billing].
 
-    The project flash space is only meant for very short term file storage.
+    It is recommended to only keep the data on /flash during the time when it's actively needed.
     
 
+Notice that there are **no** backups for any storage systems of LUMI. 
+
+Users should pay attention to make secure copy of important data elsewhere. One possibility is to keep a backup of data in LUMI-O. 
+
+
```

Comment:
I kind of would like to mention LUMI-O in this context. But if you think it could be phrased better, suggest something

Reply:
I meant to change only the first part of the sentence; lumi-o is fine.